app/src/main/java/org/groundplatform/android/model/SurveyListItem.kt

```diff
@@ -16,12 +16,15 @@
 
 package org.groundplatform.android.model
 
+import org.groundplatform.android.proto.Survey.DataVisibility
+
 data class SurveyListItem(
   val id: String,
   val title: String,
   val description: String,
   val availableOffline: Boolean,
+  val dataVisibility: DataVisibility?,
```

Comment:
This variable is not being used anywhere.

Reply:
+1. We can remove it here.